modules/ROOT/pages/customsidebar.adoc

```diff
@@ -78,6 +78,10 @@ tinymce.init({
       icon: 'comment',
       onSetup: (api) => {
         console.log('Render panel', api.element());
+	api.element().focus();
```

Comment:
What is the purpose of `api.element().focus()` since the code puts the focus back on the editor on return?

Reply:
I assume that the `api.element().focus()` is to put focus on the sidebar when it is shown on `init`